tests/testthat/_snaps/print.md

```diff
@@ -51,8 +51,8 @@
       <https://doi.org/10.1016/S1473-3099%2809%2970069-8>.
       Distribution: lnorm
       Parameters:
-        meanlog: 0.660
-        sdlog: 1.205
+        meanlog: 1.386
+        sdlog: 0.593
```

Comment:
Are these supposed to change?

Reply:
These are changing due to a bug fix in #378 which are now updating the automatically calculated parameters of these database entries, so this change is expected.